builder/ConfTargets.cmake

```diff
@@ -23,6 +23,8 @@ message( STATUS "Global Configuration of Targets" )
 set( T_ARCH "sse4.2" )
 message( STATUS "Target Architecture to compile: ${T_ARCH}" )
 
+set( CMAKE_ASM_FLAGS_RELEASE "-O3" )
```

Comment:
was that a warning fix as well? which?

Reply:
CMAKE_ASM_FLAGS_RELEASE contains "-O3 -DNDEBUG" by default.
And now during compilation assembly files clang says that "-D NDEBUG" option is unused.

Reply:
As a major rule: you can _not_ override potential user input. And you are doing exactly that by this `set()`.

I think we were handling this issue somehow - please, check with our CI how we actually call cmake for clang. In any case - please, drop this change. Let's focus on other issues rather than this one.

Reply:
Ok, got it